cmd/frontend/internal/app/jscontext/jscontext.go

```diff
@@ -75,6 +76,16 @@ type UserSession struct {
 	CanSignOut bool `json:"canSignOut"`
 }
 
+type Permission struct {
+	Typename    string     `json:"__typename"`
```

Comment:
nit: for the sake of naming consistency :)
```suggestion
	GraphQLTypename    string     `json:"__typename"`
```